### PR TITLE
Support Azure OpenAI as a LLM source

### PR DESCRIPTION
**Pull request by olewicki**

### CODE REVIEW

File: bugbug/generative_model_tool.py [hunk 1]
```diff
@@ -7,19 +7,28 @@
...
 
 from langchain_community.llms import HumanInputLLM
-from langchain_openai import ChatOpenAI
+from langchain_openai import AzureChatOpenAI, ChatOpenAI
 
 from bugbug.utils import get_secret
 
 
 def create_llm(llm):
+    temperature = 0.2
```
Comment (suhaibmujahid) [inline bugbug/generative_model_tool.py:16]:
```suggestion
    openai_temperature = 0.2
```

Comment (olewicki) [reply bugbug/generative_model_tool.py]: fixed

Comment (suhaibmujahid) [reply bugbug/generative_model_tool.py]: It seems that you added some unrelated changes, which were not there when you opened the PR. I guess you copied commits from the master branch.

**Comment by suhaibmujahid** [comment]
Closed in favour of https://github.com/mozilla/bugbug/pull/4263

**Pull request closed without merging**